crates/bevy_reflect/bevy_reflect_derive/src/derive_data.rs

```diff
@@ -54,6 +56,7 @@ pub(crate) struct ReflectMeta<'a> {
 /// ```
 pub(crate) struct ReflectStruct<'a> {
     meta: ReflectMeta<'a>,
+    serialization_blacklist: BitSet<u32>,
```

Comment:
```suggestion
    serialization_denylist: BitSet<u32>,
```

Reply:
Ditto elsewhere :) I think it's clearer and I've met people the term bothered.

Reply:
Understandable! Force of habbit, was the only term I had for the concept, thanks!